snap-wrappers/bin/launch-multipass

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+. client-common.sh
+link_autostart
```

Comment:
I wonder if we should be doing this for the CLI, too?

Reply:
My reasoning is: the CLI is our interface to the autostart flag; without this, `multipass set client.gui.autostart=true` wouldn't work (until the gui was manually started); the clients share the same environment (same settings file even), so whichever comes up first can get the required infrastructure in place.

Reply:
OK true.